### DISCUSSION TITLE
Edge extension installation causes an error

### DISCUSSION THREAD

**Discussion by kakkun61** [aclap-dev/video-downloadhelper#1487] [Help and Q&A] [answered]
### Your question

I tried to install the Edge extension but an error happens.

- extension URL
  - https://microsoftedge.microsoft.com/addons/detail/video-downloadhelper/jmkaglaafmhbcpleggkmaliipiilhldn
- error message on UI
  - > An error has occurred
    > There was a problem getting the manifest file for this add-on. Please try again.
- error message on console of developer tool
  - > Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'strict-dynamic' 'nonce-AkME4jRt9GzcXRkrA2NLvMjqkNpWQqu9S1W2q50C3Qs=' 'unsafe-inline'". Note that 'unsafe-inline' is ignored if either a hash or nonce value is present in the source list.
  - File: https://microsoftedge.microsoft.com/addons/detail/video-downloadhelper/jmkaglaafmhbcpleggkmaliipiilhldn 
  - Line: 826

- Microsoft Edge version 131.0.2903.51

### Addon & CoApp version and details

Not yet installed addon nor coapp

**Comment by jcfrog** [comment]
Hi,
Thanks for the feedback, we'll have a look at this.

**Comment by jcfrog** [comment, marked as answer]
@kakkun61 it seems the issue is gone. Probably a bug on Microsoft side. 
Just in case, do you still have the problem?

**Reply by kakkun61** [reply]
I have no problem now! Mmmmicrosoft😳
